### PR TITLE
Debian10 fix

### PR DESCRIPTION
**Pull request by gery6**
Fix $sentinel_pid_file  for Debian.
Systemd will lookup for the pidfile in /var/run/sentinel/redis-sentinel.pid

#### Pull Request (PR) description
Debian10 default path for pidfile of sentinel will -> /var/run/sentinel/redis-sentinel.pid

#### This Pull Request (PR) fixes the following issues
#322

### CODE REVIEW

**Comment by gery6** [comment]
@dhoppe: You're right. Debian Stretch has another path. 
I will spend some work on it, soon, when the time will allows it.

Thank you.

**Comment by basti-nis** [comment]
I got the same problem, but your new pid file path won't work for me.

> Distributor ID: Debian
Description:    Debian GNU/Linux 10 (buster)
Release:        10
Codename:       buster
Debian 10.2


> redis-sentinel:
  Installed: 5:5.0.3-4+deb10u1
  Candidate: 5:5.0.3-4+deb10u1
  Version table:
 *** 5:5.0.3-4+deb10u1 500


The pid file path for me is as follows:
`/run/sentinel/redis-sentinel.pid`

After i changed the parameter to:

`pid_file => '/run/sentinel/redis-sentinel.pid'`

It worked like a charm.


Seems there is no consistency.


Which versions do you use?

File: manifests/params.pp [hunk 1]
```diff
@@ -23,7 +23,13 @@
       $sentinel_daemonize        = true
       $sentinel_init_script      = '/etc/init.d/redis-sentinel'
       $sentinel_package_name     = 'redis-sentinel'
+
+      if versioncmp($facts['os']['release']['full'], '10.0.0') >= 0 {
+      $sentinel_pid_file         = '/var/run/sentinel/redis-sentinel.pid'
```
Comment (ekohl) [inline manifests/params.pp]:
```suggestion
      $sentinel_pid_file         = '/run/sentinel/redis-sentinel.pid'
```

Comment (ekohl) [reply manifests/params.pp]: This was marked as resolved, but I don't see the actual change so I've marked it as unresolved. Mind having a look?

File: spec/classes/redis_sentinel_spec.rb [hunk 2]
```diff
@@ -10,6 +10,8 @@
           '/etc/redis/redis-sentinel.conf.puppet'
         when 'Debian'
           '/etc/redis/redis-sentinel.conf.puppet'
+        when 'Ubuntu'
```
Comment (ekohl) [inline spec/classes/redis_sentinel_spec.rb]: There is no Ubuntu OS family since Ubuntu is part of the Debian OS family.

**Pull request closed without merging**